### DISCUSSION TITLE
ConvexSpheropolyhedronUnion

### DISCUSSION THREAD

**Discussion by Harsh-hkr** [glotzerlab/hoomd-blue#1945] [General]
Hi all, 

I am relatively new to Hoomd blue and was trying to experiment with running simulations on a shape that can be defined with 2 adjoint cubes, one larger than the other. I have tried to use ConvexSpheropolyhedronUnion according to the documentation however fail to output the results I seek. 

Is there any tutorial for this integrator or working code that I am able to refer to in order to help me set this up? Thank you!

**Comment by joaander** [comment]
There is no tutorial specific to `ConvexSpheropolyhedronUnion`. There are a few examples of valid shape definitions in the unit tests suite: https://github.com/glotzerlab/hoomd-blue/blob/ec0b84cdf49987f9d6fdd5d1f6d4bfa216edbfc7/hoomd/hpmc/pytest/conftest.py#L357-L409
